static/templates/settings/organization_settings_admin.hbs

```diff
@@ -141,6 +141,12 @@
                   reset_button_text=(t '[Unset]')
                   label=admin_settings_label.realm_default_code_block_language }}
 
+                {{> settings_checkbox
+                  setting_name="realm_enable_spectator_access"
+                  prefix="id_"
+                  is_checked=realm_enable_spectator_access
+                  label=admin_settings_label.realm_enable_spectator_access}}
```

Comment:
This probably should only appear if the server-level option to allow this is enabled.

Reply:
OK did that via `render_only=page_params.development_environment`, though I think we probably want that to be more like `render_only=page_params.server_web_public_streams_enabled` after adding that property to the API?

Reply:
The intent is that Zulip Cloud would enable the server-level `WEB_PUBLIC_STREAMS_ENABLED`, as would self-hosters running open source projects, but most self-hosted Zulip servers would not.  (This is a mechanism to help protect corporate Zulip servers from accidentally having web-public content).